docs/apidiffs/current_vs_latest/opentelemetry-extension-trace-propagators.txt

```diff
@@ -1,2 +1,17 @@
 Comparing source compatibility of  against 
-No changes.
\ No newline at end of file
+***! MODIFIED CLASS: PUBLIC FINAL io.opentelemetry.extension.trace.propagation.B3ConfigurablePropagator  (not serializable)
```

Comment:
Oops - sorry I didn't run a full build on the previous PR

Reply:
shame!